.github/workflows/publish-packages-1.0.yml

```diff
@@ -73,13 +73,51 @@ jobs:
 
             if ($firstPackageVersion -eq '')
             {
-              $firstPackageVersion = $packageVersion
+                $firstPackageVersion = $packageVersion
             }
 
+            $changelogContent = Get-Content -Path "src/$packageName/CHANGELOG.md"
+
+            $headingWritten = $false
+            $started = $false
+            $content = ""
+
+            foreach ($line in $changelogContent)
+            {
+                if ($line -like "## ${{ github.ref_name }}" -and $started -ne $true)
+                {
+                    $started = $true
+                }
+                elseif ($line -like "Released *" -and $started -eq $true)
```

Comment:
Don't you think, adding this to the content would help?

Reply:
I don't think it is really necessary because the release itself where this is presented has the same date. For something like `core-` tag there are a lot of packages pushed so this line will be repeated like 9/10 times too in the content.